contracts/script/utils/Utils.sol

```diff
@@ -105,4 +105,15 @@ contract Utils is Script {
         );
         vm.writeJson(outputJson, outputFilePath);
     }
+
+    function _record_addresses(
```

Comment:
i don't see this method called anywhere

Reply:
Yes, that was copied over from rolldown-contracts incase I'd need it - but turns out I don't, so removed it now, Thanks